package.json

```diff
@@ -32,6 +32,7 @@
     "@commitlint/cli": "^17.7.2",
     "@commitlint/config-conventional": "^17.7.0",
     "@rollup/plugin-typescript": "^8.5.0",
+    "@testing-library/dom": "^9.3.3",
```

Comment:
I think this is unnecessary. The necessary dependencies are already there in the `@testing-library/react`, and the reason eslint happens is because of type dependencies.